src/s1reader/utils/plot_bursts.py

```diff
@@ -0,0 +1,140 @@
+                        help="Directory containing orbit files")
+    parser.add_argument('-x', '--x-spacing', type=float, default=5,
+                        dest='x_spacing',
+                        help='Spacing of the geogrid in x direction')
+    parser.add_argument('-y', '--y-spacing', type=float, default=10,
+                        dest='y_spacing',
+                        help='Spacing of the geogrid in y direction')
+    parser.add_argument('-e', '--epsg', type=int, dest='epsg',
+                        help='EPSG for output coordinates')
+    parser.add_argument('-o', '--output', type=str, default='burst_map.gpkg',
+                        dest='output',
+                        help='Output filename for burst map')
+    return parser.parse_args()
+
+
+def burst_map(slc, orbit_dir, x_spacing,
+              y_spacing, epsg,
+              output_filename):
+
+    # Initialize dictionary that will contain all the info for geocoding
+    burst_map = {'burst_id':[], 'length': [], 'width': [],
+                 'spacing_x': [], 'spacing_y':[], 'min_x': [],
+                 'max_x': [], 'min_y': [], 'max_y': [], 'first_valid_line': [],
+                 'last_valid_line':[], 'first_valid_sample':[], 'last_valid_sample':[],
+                 'border':[]}
+    i_subswath = [1, 2, 3]
+    pol = 'vv'
+    orbit_list = glob.glob(f'{orbit_dir}/*EOF')
+    orbit_path = get_orbit_file_from_list(slc, orbit_list)
+
+    for subswath in i_subswath:
+        ref_bursts = load_bursts(slc, orbit_path, subswath, pol)
+        for burst in ref_bursts:
+            burst_map['burst_id'].append(burst.burst_id)
+            burst_map['length'].append(burst.shape[0])
+            burst_map['width'].append(burst.shape[1])
+            burst_map['spacing_x'].append(x_spacing)
+            burst_map['spacing_y'].append(y_spacing)
+            burst_map['first_valid_line'].append(burst.first_valid_line)
+            burst_map['last_valid_line'].append(burst.last_valid_line)
+            burst_map['first_valid_sample'].append(burst.first_valid_sample)
+            burst_map['last_valid_sample'].append(burst.last_valid_sample)
+
+            poly = burst.border[0]
+            # Give some margin to the polygon
+            margin = 0.001
+            poly = poly.buffer(margin)
+            burst_map['border'].append(Polygon(poly.exterior.coords).wkt)
+
+            # Transform coordinates from lat/long to EPSG
+            llh = osr.SpatialReference()
+            llh.ImportFromEPSG(4326)
+            tgt = osr.SpatialReference()
+
+            tgt_x, tgt_y = [], []
+            x, y = poly.exterior.coords.xy
+            tgt.ImportFromEPSG(int(epsg))
+            trans = osr.CoordinateTransformation(llh, tgt)
+            for lx, ly in zip(x, y):
+                dummy_y, dummy_x, dummy_z = trans.TransformPoint(ly, lx, 0)
+                tgt_x.append(dummy_x)
+                tgt_y.append(dummy_y)
+            
+            if int(epsg)==4326:
```

Comment:
```suggestion
            if epsg == 4326:
```
Parser already provides `epsg` as `int`.
Also formatting nitpick: `==` to ` == `.